Core/Security/X509Certificate.py

```diff
@@ -92,7 +123,7 @@ def getSubjectDN( self ):
     """
     if not self.__valid:
       return S_ERROR( DErrno.ENOCERT )
-    return S_OK( self.__certObj.get_subject().one_line() )
+    return S_OK( str( self.__certObj.get_subject() ) )
```

Comment:
It's a name object. And I wanted to keep the old interface, where getSubjectDN returned string.